### DISCUSSION TITLE
Is it possible to set author URL for a blog post?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#3405] [Q&A] [answered]
Hi,
in my `_metadata.yml` I have set

```
author:
  - name: Name Surname
    url: https://twitter.com/namesurname
```

Is this the right way to do it?
I have seen something similar in action in this [great blog](https://blog.djnavarro.net/posts/2022-11-03_what-i-know-about-mastodon/), I have explored the [repo code](https://github.com/djnavarro/quarto-blog), but I do not find the way to replicate it. I have simply the name without any link.

Thank you

**Comment by mcanouil** [comment, marked as answer]
The blog you are referencing was rendered using Quarto v1.1.149.
In v1.2, Quarto team made some changes in the author yaml scheme, thus it might be a bug.

To confirm that, could you share a reproducible example? (I could later clone the repo you mentioned and try with v1.3, or you could do it and update your post accordingly)

**Reply by aborruso** [reply]
Thank you 

> To confirm that, could you share a reproducible example? (I could later clone the repo you mentioned and try with v1.3, or you could do it and update your post accordingly)

It's still a very draft and under costruction:
- a sample URL https://aborruso.github.io/til/filtrare-file-elenco-file-esterno/
- the metadata for that folder https://github.com/aborruso/aborruso.github.io/blob/main/til/_metadata.yml

**Reply by dragonstyle** [reply]
This is a bug in Quarto - it turns out the link was only being output when an affiliation was present. I've fixed this and the next release of Quarto 1.3 will contain this fix. Apologies and thank you for catching this!

**Reply by aborruso** [reply]
> This is a bug in Quarto - it turns out the link was only being output when an affiliation was present. I've fixed this and the next release of Quarto 1.3 will contain this fix. Apologies and thank you for catching this!

thank you @dragonstyle and @mcanouil
